plugins/generator-1.19.4/forge-1.19.4/utils/mcitems.ftl

```diff
@@ -301,6 +301,8 @@
             <#else>
                 <#return '{ "Name": "' + mapped + '" }'>
             </#if>
+        <#else>
+        <#assign mapped = "minecraft:" + mapped />

```

Comment:
This changes the way mappedMCItemToBlockStateJSON behaves. Check the code below, if it is tag, it returns air as one can't convert tag to blockstate, this is antipattern

Reply:
I'm not sure to understand what you mean. The generated code is exactly how it should behave
![image](https://user-images.githubusercontent.com/38427877/236708914-fa7cd64c-0e3e-41d8-a7bd-92ae474dbdc3.png)

The first is supposed the Stone element grouping all Minecraft's stones, the second is a tag with the text "logs" written by the user and the last one is a Minecraft block.
I also tested with a custom block and the generated code is fine.

Reply:
Gnerated code for your case is ok, but this changes the way mappedMCItemToBlockStateJSON behaves.

mappedMCItemToBlockStateJSON is not meant to be used for tags, as tag can't be converted to tag